modules/images/image-loading-optimization/storage/data.php

```diff
@@ -0,0 +1,311 @@
+<?php
+/**
+ * Metrics storage data.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets the freshness age (TTL) for a given page metric.
+ *
+ * When a page metric expires it is eligible to be replaced by a newer one if its viewport lies within the same breakpoint.
+ *
+ * @return int Expiration TTL in seconds.
+ */
+function ilo_get_page_metric_freshness_ttl() {
+	/**
+	 * Filters the freshness age (TTL) for a given page metric.
+	 *
```

Comment:
add `@since` tag for the filter (? we do in core)

Reply:
Yeah, all of the functions and filters will need `@since` tags eventually. I guess I could add `@since n.e.x.t` everywhere now.

Reply:
`@since` tags added in 19ce752